### ISSUE TITLE
Registration with email and activation

### ISSUE THREAD

**Issue by catarata02** [rootzoll/konfetti-app#41]
Otherwise non-valid email or worse: from other person can be used

**Comment by rootzoll** [comment]
thats correct - but not critical for the first test runs in trusted environment. needs to ready and tested until end of year.

**Comment by catarata02** [comment]
thats not what i meant with this issue (uniqueness).. but validation by sending activation email to the user.. without sending email activation, i can still use somebody else's email..

**Comment by rootzoll** [comment]
still important, but because this needs also changes in the client user communication and first test run will be mostly mobile app users, I think we move this to next milestone.
